### PR TITLE
Whatsapp complete functionality

### PR DESCRIPTION
**Pull request by watchoutfreedom**

### CODE REVIEW

File: web/src/components/button/CardButton/index.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export default function CardButton({ button, buttonTypes }) {
             button={button}
             buttonTypes={buttonTypes}
           />
-          <ShowPhone user={button.owner} />
+          {/* <ShowPhone user={button.owner} /> */}
```
Comment (albjeremias) [inline web/src/components/button/CardButton/index.tsx:99]: remove comments please

File: web/src/components/button/CardButton/index.tsx [hunk 2]
```diff
@@ -694,3 +622,86 @@ function isButtonOwner(sessionUser, button) {
...
   );
 }
+
+export function ButtonOwnerPhone({ user }) {
+  const [showPhone, toggleShowPhone] = useState(false);
+  const [phone, setPhone] = useState(null);
+  const isLoadingPhone = React.useRef(false)
+
+
+  // const getPhone = () => {
```
Comment (albjeremias) [inline web/src/components/button/CardButton/index.tsx:632]: remove comments

File: web/src/components/map/Map/HexagonExploreMap.tsx [hunk 3]
```diff
@@ -131,7 +89,7 @@ export default function HexagonExploreMap({
             tileType={exploreSettings.tileType}
             handleClick={onMapClick}
           >
-            <HbMapOverlay selectedNetwork={selectedNetwork} />
+            {/* <HbMapOverlay selectedNetwork={selectedNetwork} /> */}
```
Comment (albjeremias) [inline web/src/components/map/Map/HexagonExploreMap.tsx:92]: why is this commented now? has nothing to do with this issue

**Review by albjeremias — CHANGES_REQUESTED** [review]

**Review by watchoutfreedom — COMMENTED** [review]
ready

**Pull request closed without merging**